src/BoxColor.js

```diff
@@ -0,0 +1,16 @@
+import './BoxColor.css'
+
+const BoxColor = (props) => {
+
+    const { r, g, b } = props
+    const background = {
+        backgroundColor: `rgb(${props.r},${props.g},${props.b})`
+    }
+    return (
+        <div>
+            <p className="box-color" style={background} >rgb({props.r},{props.g},{props.b}) </p>
+        </div>
+    )
+}
```

Comment:
Si lo deconstruyes usalo muchacho!

```suggestion
const BoxColor = (props) => {

    const { r, g, b } = props
    const background = {
        backgroundColor: `rgb(${r},${g},${b})`
    }
    return (
        <div>
            <p className="box-color" style={background} >rgb({r},{g},{b}) </p>
        </div>
    )
}
```